src/OpenTok/OpenTok.php

```diff
@@ -255,6 +255,103 @@ public function createSession($options = array())
         ));
     }
 
+    /**
+     * This method starts an Experience Composer Render session.
```

Comment:
Please clone the REST API docs (with modifications):

```suggestion
     * Starts an Experience Composer renderer for an OpenTok session.
     * For more information, see the
     * <a href="https://tokbox.com/developer/guides/experience-composer">Experience Composer
     * developer guide</a>.
```